src/Refactoring-Core/RBParameterRemoval.class.st

```diff
@@ -0,0 +1,60 @@
+"
+I am a common superclass for `InlineParameter` and `RemoveParameter` refactoring.
+I have the common state and methods that are shared between the two refactorings.
+"
+Class {
+	#name : 'RBParameterRemoval',
```

Comment:
Shouldn't the class ends with 'Refactoring' to be consistent with names?

Reply:
I was unsure about this. This one is abstract and will never be used as refactorings, it's just a class that holds state and behavior that are used in both subclasses. But, I've renamed it so should be ready to go now. Thanks!